docs/source/terms_and_model.rst

```diff
@@ -216,6 +201,53 @@ amino acid codes.
   AA).
 
 
+.. _integerrange:
+
+IntegerRange
+############
+
+**Computational Definition**
+
+An pair of integer values used to specify an inclusive range.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "IntegerRange"
+   * - min
+     - int
+     - 0..1
+     - minimum value; inclusive
+   * - end
+     - int
+     - 0..1
+     - maximum value; inclusive
+
+**Implementation Guidance**
+
+* At least one of min or max must be specified.
```

Comment:
I'm not sure how only one would be specified? If only the min is specified does that mean the max is unbounded and vice versa? I think we should also state that they can be equal and should be when it is a specific quantity. Also we should state that min should never be greater than max and vice versa.

Reply:
I added a statement to clarify. Please resolve if it passes muster.